usaon_benefit_tool/templates/base.html

```diff
@@ -53,6 +53,7 @@
     <div class="footer-items">
       <p>&copy; USAON &amp; NSIDC</p>
       <p>Version: v{{ __version__ }}</p>
+      <a href="{{url}}"><img src="/static/github.svg" alt=""></a>
```

Comment:
Here, can you please use `render_icon('github')` to render the icon? That way we don't need to add the icon to the repo! [Here's another template that's using `render_icon`](https://github.com/nsidc/usaon-benefit-tool/blob/c5c3d1164c8ca1670bba11c474025d07ec3a2abc/usaon_benefit_tool/templates/login.html#L2); don't forget the import! :)